Server/Components/Pawn/utils.hpp

```diff
@@ -852,7 +857,7 @@ inline bool GetCurrentWorkingDirectory(std::string& result)
 #endif
 inline void NormalizeScriptName(std::string name, std::string& result)
```

Comment:
One last thing in here, rename it to `Normalise` since the entire code base is based on UK English, then I'll do a full review, thanks